content/user-guide/index.md

```diff
@@ -5,7 +5,7 @@ description: How to run Mysterium dVPN on Linux
 
 ## Introduction 
 
-Mysterium node comes with a built-in command line tool `myst cli`
+Mysterium node comes with a built-in command line tools 
```

Comment:
```suggestion
Mysterium node comes with a built-in command line interface [CLI](https://en.wikipedia.org/wiki/Command-line_interface)
```

Reply:
Bu then again I have some doubts about calling it a CLI

Reply:
Thing it we got sort of two ways to interact with node `cli` or `account/connection` commands. So we should probably describe something like that in here aswell.